specs/refactoring_of_node_monitoring_flows_into_node_agent.adoc

```diff
@@ -0,0 +1,181 @@
+// vim: tw=79
+
+= Refactoring Of Node Monitoring Flows Into Node-Agent
+
+== Introduction
+
+This specification deals with detailing the work involved in merging the
+node-monitoring and node-agent services.
+
+Currently the responsibilities of node-monitoring are as follows:
+
+* It packages the collectd custom plugins
```

Comment:
@r0h4n Yes for now its going to be as you said..
My comment was to handle the case when we move this gluster space..
Which is when gluster does not know how and what cluster id is which is why the problem then is what happens to cluster-id part of metric namespace.. So, we'll need to keep the prefix part(tendrl.clusters.cluster-id or tendrl.cluster.) of namespace as configurable(only to the extent of handling cases of presence/absence of cluster-id) but may not be for now :) ..